scripts/build/win32/build.py

```
@@ -15,38 +15,30 @@
     DEV_HOME + "\include\php",
     DEV_HOME + "\include\php\main",
     DEV_HOME + "\include\php\TSRM",
-    DEV_HOME + "\include\php\Zend"
+    DEV_HOME + "\include\php\Zend",
```

Ruff (0.16.2)

[warning] Invalid escape sequence: `\i`

Use a raw string literal

(W605)